authentication_authorization/README.md

```diff
@@ -145,8 +145,8 @@ We decided to rely on django to handle login, lets see if Django can also handle
 Done reading? You should by now think about adding a url (in `mysite/urls.py`) pointing to the `django.contrib.auth.views.logout` view. Like this:
 
 ```python
-from django.conf.urls import include, url
-include django.contrib.auth.views
+from django.conf.urls import include, url, patterns
+import django.contrib.auth.views
```

Comment:
Do we want to make this `from django.contrib.auth import views`

and then modify the code on line 156 & 157 to be

```
views.login
view.logout
```

Instead?